FreeRTOS_ARP.c

```diff
@@ -99,6 +113,18 @@ static TickType_t xLastGratuitousARPTime = ( TickType_t ) 0;
     MACAddress_t xARPClashMacAddress;
 #endif /* ipconfigARP_USE_CLASH_DETECTION */
 
+/** @brief This local variable is used to keep track of number of ARP requests sent and
+ * also to limit the requests to arpIP_CLASH_MAX_RETRIES per arpIP_CLASH_RESET_TIMEOUT_MS
+ * period. */
+static UBaseType_t uxARPClashCounter = 0;
```

Comment:
Maybe write zero with a `U` suffix?
The unsigned suffix is missing in 3 locations related to `uxARPClashCounter`. Also I would suggest to define `arpIP_CLASH_MAX_RETRIES` as `1U` to avoid an extra cast in
    `if( uxARPClashCounter < arpIP_CLASH_MAX_RETRIES )`.